src/routes/publish.js

```diff
@@ -26,6 +27,7 @@ export function publishRoutes() {
 
   const router = new Router();
 
+  router.use(versioning('1.0'));
```

Comment:
I decided to call `versioning` in this file, instead of a file higher in the route hierarchy, because when we'll want to implement upgrading we'll probably want to have a different mechanism for each API.

Open question: `1.0` or `1` ? I think I'd rather use `1` for simplicity, what do you think?